pkg/apis/logging/v1/output_types.go

```diff
@@ -124,15 +124,48 @@ type Syslog struct {
 	// +optional
 	TrimPrefix string `json:"trimPrefix,omitempty"`
 
-	// TagKey specifies a record field  to  use as tag.
+	// Tag specifies a record field  to  use as tag.
 	//
 	// +optional
-	TagKey string `json:"tagKey,omitempty"`
+	Tag string `json:"tag,omitempty"`
 
 	// PayloadKey specifies record field to use as payload.
 	//
 	// +optional
 	PayloadKey string `json:"payloadKey,omitempty"`
+
+	// Rfc specifies the rfc to be used for sending syslog
+	//
+	// Rfc values can be one of:
+	//  - RFC3164 (https://tools.ietf.org/html/rfc3164)
+	//  - RFC5424 (https://tools.ietf.org/html/rfc5424)
+	//
+	// If unspecified, RFC5424 will be assumed.
+	//
+	// +kubebuilder:validation:Enum:=RFC3164;RFC5424
```

Comment:
add `// +kubebuilder:validation:Default=...` here to make the default clear.

Reply:
added